### PR TITLE
Performance: Add links to downsampling tutorials

### PR DESCRIPTION
**Pull request by amotl**
## About

Add two more references to tutorials on the community forum. Thanks, @hlcianfagna and @hammerhead.

- Performance: Add link to "Resampling time-series data with DATE_BIN"
- Performance: Add link to "Advanced downsampling with the LTTB algorithm"

## Preview

- https://cratedb-guide--45.org.readthedocs.build/performance/selects.html#downsampling-with-date-bin
- https://cratedb-guide--45.org.readthedocs.build/performance/selects.html#downsampling-with-lttb

/cc @surister, @karynzv, @marijaselakovic, @seut, @wierdvanderhaar

### CODE REVIEW

File: docs/performance/selects.rst [hunk 1]
```diff
@@ -110,6 +110,63 @@ to the original result.
...
+
+.. code-block:: sql
+
+    WITH downsampleddata AS
+    (	SELECT lttb_with_parallel_arrays(
+        array(SELECT n FROM demo ORDER BY n),
+        array(SELECT reading FROM demo ORDER BY n)
+        ,100) AS lttb)
+    SELECT unnest(lttb['0']) as n, unnest(lttb['1']) AS reading
+    FROM downsampleddata;
```
Comment (amotl) [inline docs/performance/selects.rst:167]: This probably also has a formatting hiccup, @hammerhead?

Comment (amotl) [reply docs/performance/selects.rst:167]: Reformatted using https://sqlformat.org/, then edited manually a bit. Is it a good choice?
```suggestion
    WITH downsampleddata AS
      (SELECT lttb_with_parallel_arrays(
      array(SELECT n FROM demo ORDER BY n),
      array(SELECT reading FROM demo ORDER BY n), 100) AS lttb)
    SELECT unnest(lttb['0']) AS n,
           unnest(lttb['1']) AS reading
    FROM downsampleddata;
```

Comment (hammerhead) [reply docs/performance/selects.rst:167]: I would love to have a standard formatting tool that we can agree on. Like (at least in our team) we settled on [black](https://pypi.org/project/black/) for Python, and it's considered the truth™️ that we all stick to. Without the "then edited manually a bit" part you mentioned, as that kind of defeats the purpose.

For community posts, I don't dare to be the one nit-picking on formatting all the time, but as soon as we are talking about the main documentation, there needs to be a consistent style and quality of formatting. With formatting being such a subjective topic, only a tool we all abide to can do that job.

Comment (amotl) [reply docs/performance/selects.rst:167]: I agree we need to level up. Future improvements, also expanding into cratedb-examples, will/should/may probably include:
- Using `ruff` across the board.
- Use `black[jupyter]` for canonicalizing all the Jupyter Notebooks we introduced recently.
- For prose, we are already using [Vale](https://github.com/errata-ai/vale) and [proselint](https://pypi.org/project/proselint/).
- More potential linters: https://markuplint.dev/, https://textlint.github.io/
- For other file formats, https://github.com/super-linter/super-linter has been probed at https://github.com/crate/infrastructure/pull/2688.
- For SQL, the formatter at https://sqlformat.org/ is actually from [sqlparse](https://pypi.org/project/sqlparse/), which we are already using at other places.

_=> We probably want to break this out into a different discussion._

Comment (amotl) [reply docs/performance/selects.rst]: I've diverted this into another issue.

- https://github.com/crate/tech-writing/issues/396

**Comment by amotl** [comment]
Thanks for approving. Because this PR is stacked upon GH-42, that one also needs to be approved.

**Review by hlcianfagna — APPROVED** [review]

**Pull request merged**